src/software/ai/hl/stp/play/stop_play_test.cpp

```diff
@@ -12,7 +12,7 @@
 
 class StopPlayTest : public SimulatedPlayTestFixture
 {
-   public:
+   protected:
```

Comment:
I'm curious what the benefit of using `protected` here is instead of `public`?

Reply:
in this case, there isn't much benefit, but as a principle we should limit the access of functions as much as we can (make the interface smaller)